utxo/utxo.go

```diff
@@ -1710,10 +1710,19 @@ func (uv *UtxoVM) queryAccountContainAK(address string) ([]string, error) {
 }
 
 func (uv *UtxoVM) queryTxFromForbiddenWithConfirmed(txid []byte) (bool, bool, error) {
+	// 如果配置文件配置了封禁tx监管合约，那么继续下面的执行。否则，直接返回.
+	forbiddenContract := uv.ledger.GenesisBlock.GetConfig().ForbiddenContract
+	if len(forbiddenContract) != 1 {
```

Comment:
This requires `len(forbiddenContract) ` equal 1, but the config of `xuper.json.forbidden_contract` is an array. emm, I not really understand.

Reply:
In fact, I wanted to reuse the code. And I have corrected it.